metriq_gym/dispatch_clops.py

```diff
@@ -0,0 +1,62 @@
+"""Dispatch a CLOPS job with CLI parameters to Qiskit and wait for result."""
+import json
+import logging
+import os
+import sys
+
+from dataclasses import asdict
+
+from qiskit_ibm_runtime import QiskitRuntimeService
+
+from metriq_gym.parse import parse_arguments
+from metriq_gym.bench import BenchJobResult, BenchJobType, BenchProvider
+from metriq_gym.hardware.clops_benchmark import clops_benchmark
+
+
+logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s")
+
+def main():
+    args = parse_arguments()
+
+    if not args.token:
```

Comment:
if the account has already been saved in a previous run, the user doesn't need to pass the token